pwiz_tools/Skyline/Model/Results/ChromHeaderInfo.cs

```diff
@@ -2677,7 +2656,7 @@ public static bool IsDerivative(this TransformChrom transformChrom)
     public class ChromatogramInfo
     {
         public const double OPTIMIZE_SHIFT_SIZE = 0.01;
-        private const double OPTIMIZE_SHIFT_THRESHOLD = 0.001;
+        private const double OPTIMIZE_SHIFT_THRESHOLD = 0.0015;

```

Comment:
Why was this change necessary? Is there a particular Skyline document whose optimization chromatograms did not have the expected product m/z's?

Reply:
This was from trying to load `MRM Triggered MRM data imported.sky` in `TestExportMethodDlg`. Most of the m/zs were within the threshold, but a small percentage were very slightly over 0.001. I didn't look too much into it, I'm just assuming it was some floating point issue going on.